### PR TITLE
Alerts update

### PR DESCRIPTION
**Pull request by michalsn**
This PR addresses requests mentioned in the [#118](https://github.com/lonnieezell/forum-example/pull/118#discussion_r1377244694)

* configurable display time for alert via 3rd (optional) parameter 
```php
alerts()->set('success', 'alert message', 10);
```
* pause alert's display time on hover 
* show a progress bar for every alert (this doesn't look very good right now because I use a standard progress component, but it can be fixed in the future)

### CODE REVIEW

**Comment by michalsn** [comment]
I will rebase this PR if/when #124 will be merged.

**Comment by michalsn** [comment]
Rebased.

**Review by neznaika0 — COMMENTED** [review]
I've looked through, but I don't understand HTMX/JS yet

File: app/Filters/AlertsFilter.php [hunk 1]
```diff
@@ -26,6 +26,8 @@ public function before(RequestInterface $request, $arguments = null)
      */
     public function after(RequestInterface $request, ResponseInterface $response, $arguments = null)
     {
+        helper('alerts');
```
Comment (neznaika0) [inline app/Filters/AlertsFilter.php:29]: I have already prepared a PR)

Comment (neznaika0) [reply app/Filters/AlertsFilter.php:29]: I would remove phpdoc. It contradicts the interface. This is a question for the framework. Need to either fix `void` to `null` in the interface
```php
    /**
     * @param array|null $arguments
     *
     * @return mixed
     */
```

**Review by datamweb — APPROVED** [review]
LGTM!
@michalsn  thank you!

**Review by lonnieezell — APPROVED** [review]

**Pull request merged**